sdk/ml/azure-ai-ml/tests/automl_job/unittests/test_forecasting_job.py

```diff
@@ -84,3 +85,153 @@ def test_forecasting_task(self):
         assert isinstance(original_obj.validation_data, Input), "Validation data is not Input"
         assert original_obj.validation_data.type == AssetTypes.MLTABLE, "Test data type not set correctly"
         assert original_obj.validation_data.path == "https://foo/bar/valid.csv", "Test data path not set correctly"
+        assert original_obj.training.training_mode == None, "Training mode not set correctly"
+        assert original_obj.limits.max_nodes == 1, "Max nodes not set correctly"
+
+    def test_forecasting_task_distributed_mode(self):
+        # Create AutoML Forecasting Task
+        identity = UserIdentityConfiguration()
+        forecasting_job = forecasting(
+            training_data=Input(type=AssetTypes.MLTABLE, path="https://foo/bar/train.csv"),
+            target_column_name="target",
+            primary_metric=ForecastingPrimaryMetrics.NORMALIZED_MEAN_ABSOLUTE_ERROR,
+            enable_model_explainability=False,
+            validation_data=Input(type=AssetTypes.MLTABLE, path="https://foo/bar/valid.csv"),
+            # Job attributes
+            compute="cluster-forecasting",
+            name="forecasting_job",
+            experiment_name="foo_exp",
+            tags={"foo_tag": "bar"},
+            identity=identity,
+        )  # type: ForecastingJob
+        forecasting_job.set_limits(
+            timeout_minutes=60,
+            max_trials=100,
+            max_concurrent_trials=4,
+            max_nodes=4,
+        )
+        forecasting_job.set_training(
+            enable_onnx_compatible_models=True,
+            blocked_training_algorithms=[ForecastingModels.AVERAGE],
+            training_mode=TabularTrainingMode.DISTRIBUTED,
+        )
+        forecasting_job.set_featurization(enable_dnn_featurization=True)
+        forecasting_job.set_forecast_settings(
+            time_column_name="time",
+            forecast_horizon=20,
+            target_lags=[1, 2, 3],
+            time_series_id_column_names=["tid1", "tid2", "tid2"],
+            frequency="W-THU",
+            target_rolling_window_size=4,
+            short_series_handling_config=ShortSeriesHandlingConfiguration.DROP,
+            use_stl="season",
+            seasonality=3,
+        )
+
+        # serialize and deserialize again and compare
+        rest_obj = forecasting_job._to_rest_object()  # serialize to rest object
+        assert isinstance(rest_obj.properties.identity, RestUserIdentity)
+
+        original_obj = ForecastingJob._from_rest_object(rest_obj)  # deserialize from rest object
+        assert forecasting_job == original_obj, "Conversion to/from rest object failed"
+        assert original_obj.training.training_mode == TabularTrainingMode.DISTRIBUTED, "Training mode not set correctly"
+        assert original_obj.limits.max_nodes == 4, "Max nodes not set correctly"
+
+    def test_forecasting_task_non_distributed_mode(self):
+        # Create AutoML Forecasting Task
+        identity = UserIdentityConfiguration()
```

Comment:
Is identity required in automl test submission?

Reply:
I added it from the test above. Since it is not required, I have removed it from the newly created tests. Thanks!